src/constants/cron-job.constant.js

```diff
@@ -3,6 +3,7 @@ const TIMEZONE = 'Asia/Ho_Chi_Minh';
 const CRON_JOB_TIME = {
   SEND_BIRTHDAY_EMAIL: '0 7 * * *',
   DELETE_EXPIRED_MOMENTS: '0 7 * * *',
```

Comment:
```suggestion
  DELETE_EXPIRED_MOMENTS: '0 7 * * *',
```
update lại cài này sang 1 khung giờ khác nữa nhé